check/check_test.go

```diff
@@ -26,13 +26,13 @@ func TestCheck_Run(t *testing.T) {
 
 	ts := new(auditeval.Tests)
 	if err := yaml.Unmarshal([]byte(def1), ts); err != nil {
-		t.Fatalf("error unmarshaling tests yaml")
+		t.Fatalf("error unmarshaling tests yaml %v", err)
 	}
 
-	checkTypeManual := Check{Type: "manual", Commands: textToCommand("ps -ef"), Tests: ts, Scored: true}
-	checkTypeSkip := Check{Type: "skip", Commands: textToCommand("ps -ef"), Tests: ts, Scored: true}
-	checkNotScored := Check{Type: "", Commands: textToCommand("ps -ef"), Tests: ts, Scored: false}
-	checkNoTests := Check{Type: "", Scored: true}
+	checkTypeManual := Check{Type: "manual", Commands: textToCommand("ps -ef"), Tests: ts, Scored: true, auditer:&Audit{"ps -ef"}}
```

Comment:
Not sure why you need both `textToCommand("ps -ef")` and `ps -ef` as parameters to the same test. At the point where you call `check.Run` should `auditer` already be set up? In which case you don't need `Commands` set up here, do you?

Reply:
@igorbeider